bin/partyfy-cli.js

```diff
@@ -19,6 +19,10 @@ const main = () => {
         try {
           const imageFile = fs.readFileSync(path.resolve(source));
 
+          if (source.endsWith('.gif')) {
+            console.warn('Warning: frameDelay is currently ignored for .gif files.\n')
+          }
+
```

Comment:
We can support gifs that aren't animated or only have one frame.  This can be moved into `getFrames` and checked like this: https://github.com/dmmulroy/partyfy/pull/4/files#diff-16d18182547550b6ca24cffe5506e0a2L74-L75

Reply:
Yeah, I was thinking the same thing. So just ditch `GifUtil.read` and do it the way we handle pngs now regardless, then log a warning if we detect it's an animated gif?

Reply:
Actually using `GifUtil.read` is probably fine if we had a length check here `GifUtil.read` and log the warning there.

Reply:
Okay, I suppose we could read it with `GifUtil.read`, then if it's just one frame, mutate the result similar to how we're doing it for pngs, otherwise we just leave it the same way and log the warning?